cluster-autoscaler/utils/drain/drain.go

```diff
@@ -32,6 +32,9 @@ import (
 const (
 	// PodDeletionTimeout - time after which a pod to be deleted is not included in the list of pods for drain.
 	PodDeletionTimeout = 12 * time.Minute
+
+	// PodDeletionExtraBufferDuration - time after which a pod, that is terminating and that has run over its terminationGracePeriod, should be ignored and considered as deleted
+	PodDeletionExtraBufferDuration = 30 * time.Second
```

Comment:
To keep this in line with `IsPodLongTerminating`, maybe `PodLongTerminatingExtraThreshold`?